modules/utxo-lib/src/bitgo/zcash/ZcashPsbt.ts

```diff
@@ -128,6 +128,15 @@ export class ZcashPsbt extends UtxoPsbt<ZcashTransaction<bigint>> {
     this.tx.consensusBranchId = getDefaultConsensusBranchIdForVersion(network, version);
   }
 
+  signInput(inputIndex: number, keyPair: Signer, sighashTypes?: number[]): this {
+    (this as any).__CACHE.__UNSAFE_SIGN_NONSEGWIT = true;
```

Comment:
Maybe we should throw a comment in here about why this is safe (zcash hashes the value directly)?